dashboard/config/android-modules/zip-kernel-headers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# $1 Kernel common dir
+function zip_kernel_headers() {
```

Comment:
I think we need to call this function, similar to how we call `create_modules_staging` on line 60 of `create-initramfs.sh`.

Reply:
Makes sense, thanks! I just removed the function since its  redundant with the file name.